embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/SessionHandler.kt

```diff
@@ -138,7 +139,7 @@ internal class SessionHandler(
 
             handleAutomaticSessionStopper(automaticSessionCloserCallback)
             addFirstViewBreadcrumbForSession(startTime)
-            startPeriodicCaching { onPeriodicCacheActiveSession() }
+            startPeriodicCaching { Systrace.trace("cache-session") { onPeriodicCacheActiveSession() } }
```

Comment:
This will only measure construction of the session object, not caching (as the actual I/O happens on a background thread apart from when there's a crash)

Reply:
Yeah I should rename this. This is very fast the way it is and I just want to make sure it doesn't regress.